charts/kubescape-operator/values.yaml

```diff
@@ -461,7 +467,10 @@ nodeAgent:
     updatePeriod: 10m # duration string
     prometheusExporter: enable
     httpExporterConfig: 
-      url: http://synchronizer.kubescape.svc.cluster.local:8089/apis/v1/kubescape.io/v1/runtimealerts
+      url: http://synchronizer.svc.cluster.local:8089/apis/v1/kubescape.io/v1/runtimealerts
```

Comment:
max alerts per minute configurations?

Reply:
I [set the default](https://github.com/kubescape/node-agent/pull/257) to 100.
I'm not sure we want this number to be easily modified